### PR TITLE
November Newsletter

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

**Comment by macro1** [comment]
:+1:

File: content/posts/2015-11-02_november-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+minute lightning talks. Please comment on the meetup if you're interested in
+speaking.
+
+[Monthly Meetup]: http://www.meetup.com/pythonsd/events/226177901/
+
+
+[Django Girls Inland Empire workshop][] (djangogirls.org) <br />
+Django Girls, a non-profit that helps organize programming workshops for women,
+is holding a workshop for women who would like to learn to build websites at
+UC Riverside on December 4-5. The application period is closed but they are
```
Comment (treyhunner) [inline content/posts/2015-11-02_november-newsletter.md]: Applications are not closed.  Applications are still open!

Comment (treyhunner) [inline content/posts/2015-11-02_november-newsletter.md]: Let's just say December 5.

File: content/posts/2015-11-02_november-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+still looking for mentors.
+
+[Django Girls Inland Empire workshop]: https://djangogirls.org/inlandempire/
+
+
+[PyCon 2016][pycon-2016] (us.pycon.org) <br />
+PyCon 2016 is coming to Porland Oregon in late May 2016. Registration opens
```
Comment (treyhunner) [inline content/posts/2015-11-02_november-newsletter.md]: Registration is now open: https://us.pycon.org/2016/registration/

**Comment by treyhunner** [comment]
@davidfischer suggested a couple changes.  The Django Girls ones are important.

:+1: besides those!

**Pull request merged**